tests/unit/amo/pages/TestUserProfile.js

```diff
@@ -776,7 +777,15 @@ describe(__filename, () => {
 
     renderUserProfile({ params, store });
 
-    sinon.assert.notCalled(dispatchSpy);
+    sinon.assert.neverCalledWith(
+      dispatchSpy,
+      setUserReviews({
```

Comment:
I think you want to be asserting about `fetchUserReviews` here, not `setUserReviews`, as the test description states.